CONTRIBUTING.rst

```diff
@@ -322,69 +324,100 @@ The build status for pyiron and all sub packages are given below
     :target: https://ci.appveyor.com/project/pyiron-runner/pyiron/branch/master
     :alt: Build status
 
-pyiron releases
----------------
+pyiron release distribution
+---------------------------
 
 .. image:: https://anaconda.org/conda-forge/pyiron/badges/downloads.svg
     :target: https://anaconda.org/conda-forge/pyiron/
     :alt: Downloads
 
-For the pyiron release management we use git tags::
-
-   https://git-scm.com/book/en/v2/Git-Basics-Tagging
-
+Pyiron is released through `conda-forge`_ and  `pip`_. 
+Both packages are created automatically and maintained with every new release of pyiron. In order to use these distributions simply use the following command for conda::
+   conda install -c conda-forge pyiron
+In order to use the pip distribution use::
+   pip install pyiron
+Just like each other commit to the master branch the tagged releases are pushed to pypi.org (https://pypi.org/project/pyiron/#history)::
+The major difference for pypi (pip) is that installing pre-release versions is possible using the `--pre` flag::
+   pip install --pre pyiron
+Those pre-release versions are named `<version_number>.post0.dev<release number>` ::
+   0.2.0.post0.dev1
+For pip both the pre-releases as well as the official releases are available. For conda only the official releases are available.
+
+Building process for a release
+---------------------------------
+1. Create a Git tag to mark the release
+This step is done manually and important to trigger all the following steps. Tag can be created under https://github.com/pyiron/pyiron/tags. 
+The following steps are automated and will be performed once a tag is created. 
+In order to keep the tags consistent please follow the `Git-Tag-Guide`_.
 The tag format consists of a tag_prefix (<package name>-) and the release version, for example::
+     pyiron-0.2.0
+2. Automatically create PyPi package
+  After the tag is created, the `Deploy-Workflow`_ is triggered, which creates the PyPi Package.
+  The configuration of the release is included in the `setup.ctg file`_.
+  This Workflow first installs all dependencies, then allows for future versions of the dependencies and builds the package. After that the package is published to `pip`_.
+3. Automatically create conda-forge package
+  This release than is recognized by a conda-forge bot, which triggers a new pull request for the conda-forge package and merges automatically if all tests pass.
+  These tests are defined as `GitHub-Action-Workflows`_ and are triggered for every new pull request. More information can be found in the next chapter.
```

Comment:
The tests on conda-forge are not our unit tests.
```suggestion
```
Probably, better add this as preparation step: 
```
Check if all tests as explained in the next chapter have passed
```